server/config/questions.json

```diff
@@ -1,6 +1,43 @@
 [
     {
         "name": "Participant",
+        "text": [
```

Comment:
yeah, but I can send you specially crafted HTTP messages without a `Host` header..